<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keyboard</class>
 <widget class="QWidget" name="keyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton0">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton9">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton8">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtono">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>o</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtone">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>e</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonq">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonw">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>w</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtoni">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>i</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtont">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>t</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonp">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>p</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtondel">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonr">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>r</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonu">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>u</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonk">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>k</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtond">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>d</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtona">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtons">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonj">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>j</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtong">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>g</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonl">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonenter">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonf">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>f</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonh">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtoncomma">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonc">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>c</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonz">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonx">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonm">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonb">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>b</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonfullstop">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>160</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtony">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonv">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>n</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
